README.md

```diff
@@ -3428,7 +3428,7 @@ Other Style Guides
   - [Ben Alman](http://benalman.com/)
   - [Dmitry Baranovskiy](http://dmitry.baranovskiy.com/)
   - [nettuts](http://code.tutsplus.com/?s=javascript)
-
+  - [MMirzaArdiansyah](https://www.freecodecamp.org/)
```

Comment:
you've removed the newline between the list and the podcasts section; could you re-add it?

also, hmm - freecodecamp isn't a blog, and it doesn't seem like it's your site?

Reply:
This user has a history of spam PRs, see https://github.com/whatwg/html/pull/3171 or https://github.com/lets-blade/hello/pull/3. I've reported them.